src/components/dashboard/SendQRSummary.js

```diff
@@ -44,7 +44,23 @@ const SendQRSummary = ({ screenProps }: AmountProps, params) => {
     setProfile(profile)
   }
 
-  const confirm = async () => ((await goodWallet.isCitizen()) ? sendGD() : faceRecognition())
+  const confirm = async () => {
+    try {
+      if (await goodWallet.isCitizen()) {
+        sendGD()
+      } else {
+        faceRecognition()
+      }
+    } catch (e) {
+      log.error('Send TX failed:', e.message, e)
```

Comment:
1. dialog should be  error
2. we are not awaiting on sendGD are we gonna catch exceptions here?

Reply:
1 - done
2 - we catch it here because the "not open on send" error is returned by the goodWallet.isCitizen() method